src/qcodes/station.py

```diff
@@ -595,7 +597,7 @@ def resolve_instrument_identifier(
             level = levels[0]
             try:
                 for level in levels:
-                    instrument = checked_getattr(
+                    instrument = checked_getattr_indexed(
```

Comment:
Could we extend the docstring above with an example of what this adds

I guess something like ?

dac.channels[0]

Reply:
on it